src/redux/configureStore.js

```diff
@@ -1,11 +1,12 @@
-import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] For this activity, we want you to use `createAsyncThunk` for defining _thunk action creators_, which should be similar to this example:

```javascript
import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```

Kindly ensure to improve upon the same!

Reply:
Done. Thanks!